### DISCUSSION TITLE
Environment variables

### DISCUSSION THREAD

**Discussion by guitaripod** [wojciech-kulik/xcodebuild.nvim#147] [General]
Any way to insert env variables like in Xcode?

![CleanShot 2024-05-13 at 20 09 31](https://github.com/wojciech-kulik/xcodebuild.nvim/assets/47460844/40bb1466-c774-4fcb-b623-39c247fca6ee)

**Comment by wojciech-kulik** [comment]
Not directly from the plugin but you can:
1. Use Tuist to generate the project from the code specification.
2. Use Xcodegen to define the project in yaml file.
3. Update your project to use `xcconfig` and define variables there.

**Reply by guitaripod** [reply]
> Update your project to use xcconfig and define variables there.

I think this is the key. I just created a swift file and if target(simulator) and put it in .gitignore. `xcconfig` might be better, haven't used it before.

**Comment by wojciech-kulik** [comment]
Variables are now supported directly in the plugin.
